### PR TITLE
Audit Bulk and Async APIs

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: examples/asyncio/audit/audit_examples/log_bulk_async.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    event2 = Event(
+        message="Sign in",
+        actor="pangea-sdk",
+    )
+
+    start = time.time()
+    try:
+        await audit.log_bulk_async(events=[event1, event2], verbose=True)
+    except pe.AcceptedRequestException:
```
Comment (dbenders) [inline examples/asyncio/audit/audit_examples/log_bulk_async.py:36]: This is the happy path, I think it shouldn't be an exception

File: packages/pangea-sdk/pangea/asyncio/services/audit.py [hunk 2]
```diff
@@ -172,9 +173,83 @@ async def log_event(
...
+        sign_local: bool = False,
+        verbose: Optional[bool] = None,
+    ) -> PangeaResponse[LogBulkResult]:
+        """
+        Log an entry
+
+        Create a log entry in the Secure Audit Log.
+        Args:
+            events (List[dict[str, Any]]): events to be logged
+            verify (bool, optional): True to verify logs consistency after response.
```
Comment (dbenders) [inline packages/pangea-sdk/pangea/asyncio/services/audit.py:194]: verify was removed

**Review by dbenders — APPROVED** [review]
LGTM, only comment: verify was removed from v2/log and v2/log_async

File: examples/asyncio/audit/audit_examples/log_bulk_async.py (hunk 1 above)
Comment (pangea-andrest) [reply examples/asyncio/audit/audit_examples/log_bulk_async.py]: Fixed

**Pull request merged**